packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-wrapper.jsx

```diff
@@ -203,7 +203,6 @@ DurationWrapper.propTypes = {
     duration_unit: PropTypes.string,
     duration_units_list: MobxPropTypes.arrayOrObservableArray,
     getDurationFromUnit: PropTypes.func,
-    is_minimized: PropTypes.bool,
```

Comment:
Btw, `<DurationWrapper />` does not accept any props, it takes everything from stores. Maybe all these `propTypes` can be deleted?

Reply:
@kate-deriv  good point :)